src/strategies/rareprotocol-staked-balance/index.ts

```diff
@@ -0,0 +1,43 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+import { strategy as erc20BalanceOfStrategy } from '../erc20-balance-of';
+
+export const author = 'rareprotocol';
+export const version = '0.0.1';
+
+const abi = [
+  'function getTotalAmountStakedByUser(address _user) external view returns (uint256 amount)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const score = await erc20BalanceOfStrategy(
+    space,
+    network,
+    provider,
+    addresses,
+    options,
+    snapshot
+  );
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+  addresses.forEach((address) =>
+    multi.call(address, options.rareStakingRegistryAddress, 'getTotalAmountStakedByUser', [address])
```

Comment:
Hi @charlescrain Better to use `erc20-balance-of` strategy with `contract-call` strategy on your space, the result will be the sum of both

Reply:
@ChaituVR Thanks for pointing that out will close!